website/content/docs/connect/cluster-peering/create-manage-peering.mdx

```diff
@@ -367,11 +366,157 @@ Successfully submitted peering connection, cluster-02, for deletion
 ```
 </Tab>
 
-<Tab heading="Consul UI">
+<Tab heading="Consul UI" group="ui">
 
 In the Consul UI, click **Peers**. The UI lists peering connections you created for clusters in that datacenter.
 
 Next to the name of the peer, click **More** (three horizontal dots) and then **Delete**. Click **Delete** to confirm and remove the peering connection.
 
 </Tab>
 </Tabs>
+
+## Traffic management between peers
+
+### Redirects and failover
+
+As of Consul v1.14, you can use [dynamic traffic management](/consul/docs/connect/l7-traffic) to configure your service mesh so that services automatically failover and redirect between peers. The following examples update the [`service-resolver` config entry](/docs/connect/config-entries/) in `cluster-01` so that Consul redirects traffic intended for the `frontend` service to a backup instance in peer `cluster-02` when it detects multiple connection failures.
```

Comment:
This mentions "failover and redirect" still. Should we make this L7 traffic management?